package.json

```diff
@@ -113,63 +113,78 @@
     ".": "./lib/index.js",
     "./app": {
       "require": "./lib/app/index.js",
-      "import": "./lib/esm/app/index.js"
+      "import": "./lib/esm/app/index.js",
+      "types": "./lib/app/index.d.ts"
```

Comment:
The docs say that the `types` field should always come first in exports: https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/#package-json-exports-imports-and-self-referencing

"Note that the "types" condition should always come first in "exports"." (under the second code sample in that section)

Reply:
Good catch! Thank you!